Gruntfile.js

```diff
@@ -9,14 +9,17 @@ module.exports = function (grunt) {
     genDir: 'public_gen',
     destDir: 'dist',
     tempDir: 'tmp',
-    arch: os.arch(),
     platform: process.platform.replace('win32', 'windows'),
   };
 
   if (process.platform.match(/^win/)) {
     config.arch = process.env.hasOwnProperty('ProgramFiles(x86)') ? 'x64' : 'x86';
   }
 
+  config.arch = grunt.option('arch') || os.arch();
```

Comment:
It is used to produce tarball file name in compress.js. The command line option is used in build.go.